docs/wallet-integration-guide/external-docs/party-management/create-party.rst

```diff
@@ -1,15 +1,120 @@
 Create a Party / Address
 ========================
 
-- Create the key
-- Generate the fingerprint
-- Construct the party ID
-- Generate the three topology transactions
-- Compute multi hash
-- Sign multi-hash
-- Build signed topology transaction function
-- Load the signed transactions onto the participant
-- Authorize the PartyToParticipant transaction
-- Observe PartyToParticipant transaction function
-
-.. todo:: Add step-by-step instructions for party creation.
+Overview
+--------
+This document describes the steps required to create a new party (wallet/address) on a validator.
+Parties represent acting entites in the network and all transaction happens between one or more parties. parties comes in two forms, internal and external.
+An internal party is created and signed using the Validators own internal keys for signing (and thereby the validator operator has full control of everything that happens on the party).
+The alternative is to create an external party, an external party is similar to how node interactions happens on other networks. In this case the signing key can be held
+externally and a signature is required alongside the transaction to authorize the action. For external parties the base flow follows three steps: Prepare a transaction, sign the transaction and submit the transaction.
+
+*This document focuses on the steps required to create an external party using the Wallet SDK.*
+
+How do i quickly allocate a party?
+-----------------------------------
+Using the wallet SDK you can quickly allocate a party using the following code snippet:
+.. literalinclude:: ../examples/01-auth.ts
+   :language: js
+   :dedent:
+if you are running with a localnet setup (https://docs.dev.sync.global/app_dev/testing/localnet.html) you can use this snippet instead:
+.. literalinclude:: ../examples/01-auth-localnet.ts
+   :laguage: js
+   :dedent:
+
+
+Create the key Pair
+-------------------
+Creating key follow standard encryption practices similarly to other blockchains. The full details of supported cryptographic algorithms can be found https://docs.daml.com/canton/usermanual/security.html#common-node-keys.
+by default a **Ed25519** is used. There exists many libraries that can be used to generate such a key pair, you can do it simply with the WalletSDK using:
+
+.. code-block:: javascript
+
+   import { TopologyController } from "@splice/wallet-sdk";
+    // static method call
+    const {publicKey, privateKey } = TopologyController.createNewKeyPair()
```

Comment:
why is this part of the `TopologyController`? Generating key pairs is not really related to the Canton topology service. Later on, the guide also imports `SignTransactionHash` as a standalone function -- maybe all this functionality should be exposed via a `KeyController` thats a minimal implementation of a signing driver?